generator/features/setupLiquidityMining.ts

```diff
@@ -143,18 +146,38 @@ export const setupLiquidityMining: FeatureModule<LiquidityMiningSetup> = {
           }),
         ],
         fn: [
+          
           `
-          function test_activation() public {
-            vm.prank(EMISSION_ADMIN);
-            IEmissionManager(${pool}.EMISSION_MANAGER).configureAssets(_getAssetConfigs());
-
-            emit log_named_bytes(
-              'calldata to submit from Gnosis Safe',
-              abi.encodeWithSelector(
-                IEmissionManager(${pool}.EMISSION_MANAGER).configureAssets.selector,
-                _getAssetConfigs()
-              )
+          function buildActions() public view returns (IPayloadsControllerCore.ExecutionAction[] memory) {
+            IPayloadsControllerCore.ExecutionAction[]
+              memory actions = new IPayloadsControllerCore.ExecutionAction[](1);
+            actions[0].target = AaveV3Polygon.EMISSION_MANAGER;
```

Comment:
I would not hardcode it to `AaveV3Polygon` but specific for the network

Reply:
Oh, sure